components/src/dynamo/vllm/omni/omni_handler.py

```
@@ -288,6 +312,60 @@ def _engine_inputs_from_chat(self, request: Dict[str, Any]) -> EngineInputs:
+                    else None
+                ),
+                renderer=renderer,
+                add_generation_prompt=chat_request.add_generation_prompt,
+                continue_final_message=chat_request.continue_final_message,
+                documents=getattr(chat_request, "documents", None),
+                add_special_tokens=chat_request.add_special_tokens,
+            )
+            if not engine_prompts:
+                return None
+            return engine_prompts[0] if isinstance(engine_prompts[0], dict) else None
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)